build.sbt

```diff
@@ -4,7 +4,7 @@ lazy val sparkVersion = "2.4.0"
 
 lazy val commonSettings = Seq(
   organization := "com.google.cloud.spark",
-  version := "0.14.1-beta-SNAPSHOT",
+  version := "0.15.2-beta-SNAPSHOT",
```

Comment:
why is it 0.15.2-beta-SNAPSHOT here when it 0.15.1 in Readme?

Reply:
The README references the last released version which is 0.15.1-beta. the build.sbt has the next release version.